src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -795,6 +795,11 @@ private void DetectAttributeAndOptionConflicts(DiagnosticBag diagnostics)
                 }
             }
 
+            if (_compilation.Options.PublicSign && (object)this.AssemblyKeyContainerAttributeSetting != (object)CommonAssemblyWellKnownAttributeData.StringMissingValue)
+            {
+                diagnostics.Add(ErrorCode.WRN_AttributeIgnoredWhenPublicSigning, NoLocation.Singleton, AttributeDescription.AssemblyKeyNameAttribute.FullName);
```

Comment:
The 'if' is checking Assembly**KeyContainer**AttributeSetting, but the warning is reported for Assembly**KeyName**Attribute. Is this intentional? Are they the same attributes?